apps/site/src/app/admin/applicants/components/HackerThresholdInputs.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import axios from "axios";
+
+import {
+	Box,
+	Button,
+	Input,
+	SpaceBetween,
+} from "@cloudscape-design/components";
+
+import useHackerThresholds from "@/lib/admin/useHackerThresholds";
+
+function HackerThresholdInputs() {
+	const { thresholds } = useHackerThresholds();
+	const [acceptValue, setAcceptValue] = useState("");
+	const [waitlistValue, setWaitlistValue] = useState("");
+
+	const [status, setStatus] = useState("");
+
+	async function submitThresholds() {
+		await axios
+			.post("/api/admin/set-thresholds", {
+				accept: acceptValue,
+				waitlist: waitlistValue,
+			})
```

Comment:
Thought: Currently, the reviewer would need to input a float for both `accept` and `waitlist.` In the case that the reviewer wants to only modify one threshold, passing the old threshold for `accept` when `acceptValue` is an empty string "" or `waitlist` when `waitlistValue` is an empty string "" should keep the unchanged threshold value the same. 

In the future in a separate issue, I think it would be good to add Form validation to check to see if the thresholds are valid floats, and also Form validation to check to see if review scores are valid floats in #526